### ISSUE TITLE
Download the entire dataset

### ISSUE THREAD

**Issue by gabrielsantosrv** [aoifemcdonagh/audioset_processing#8]
First of all, I would like to thank you for your repo, it is really useful!

Is there any way to download the whole dataset?

**Comment by rafaelgreca** [comment]
I don't think so, buddy. Maybe replace the argument passed in the "-c" param for a list containing all the classes? You could try read the "class_labels_indices.csv", get all the unique values in the index/mid column (i don't know which one he uses to filter the dataset) and set it to be the default value for the "-c" param. I don't test it, but who knows? Maybe works :)
